stable/mission-control/templates/insight-server-deployment.yaml

```diff
@@ -104,33 +104,18 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-elasticsearch
               key: esPassword
-        - name: JFMC_MISSION_CONTROL_CERT
-          value: "{{ .Values.insightServer.home }}/{{ .Values.insightServer.name }}/_MASTER_/data/contexts/security/jfmc.crt"
-        - name: JFMC_INSIGHT_SERVER_CERT
-          value: "{{ .Values.insightServer.home }}/{{ .Values.insightServer.name }}/_MASTER_/data/contexts/security/insight.crt"
-        - name: JFMC_INSIGHT_SERVER_KEY
-          value: "{{ .Values.insightServer.home }}/{{ .Values.insightServer.name }}/_MASTER_/data/contexts/security/insight.key"
+        - name: JFMC_INSIGHT_SERVER_DB
+          value: "{{ .Values.mongodb.db.insightServerDb }}"
+        - name: JFIS_ALLOW_IP
+          value: "0.0.0.0/0"
```

Comment:
@amithins - please make this a configurable parameter. Can help with simple hardening.

Reply:
@amithins can you update the PR as per @eldada request, so we can them merge the PR, thanks